recursion/gnark-ffi/src/groth16.rs

```diff
@@ -139,7 +183,8 @@ impl Groth16Prover {
         }
     }
 
-    pub fn prove<C: Config>(witness: Witness<C>, build_dir: PathBuf) -> Groth16Proof {
+    // Proves by using the files in the data directory.
+    pub fn prove_local<C: Config>(witness: Witness<C>, build_dir: PathBuf) -> Groth16Proof {
```

Comment:
Just leave prove. The circuit downloading shouldn't be handled here

Reply:
removed prove_local